backend/routes.go

```diff
@@ -43,5 +43,10 @@ func newRouter(cs *server) (*httprouter.Router, error) {
 	r.PATCH("/queries/:id", handlers.CorsMiddleware(bearerMiddleware(cs.updateQueryHandler, os.Args[3])))
 	r.DELETE("/queries/:id", handlers.CorsMiddleware(bearerMiddleware(cs.deleteQueryHandler, os.Args[3])))
 
+	// event
+	r.POST("/events", handlers.CorsMiddleware(bearerMiddleware(cs.createEventHandler, os.Args[3])))
+	r.DELETE("/events/:id", handlers.CorsMiddleware(bearerMiddleware(cs.deleteEventHandler, os.Args[3])))
+	r.GET("/events", handlers.CorsMiddleware(bearerMiddleware(cs.getEventsHandler, os.Args[3])))
```

Comment:
We probably want to also have a route to get a specific event by ID

Reply:
Added a new route. It is not used from the frontend yet. Can work on that in a separate CL.

Reply:
Added a new route but it is not being used by the frontend yet. Can work on that in a separate PR.